### PR TITLE
br: remove s3 region param

### PR DESCRIPTION
**Pull request by WangLe1321**
Signed-off-by: WangLe1321 <wangle1321@163.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Remove s3.region param in br/lightning/dumpling

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
@WangLe1321 The deleted `region` parameter also exists in other files. Should we delete it from these files as well?
image

File: br/backup-and-restore-storages.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ S3、 GCS 和 Azblob 等云存储有时需要额外的连接配置，你可以
 
     ```bash
     ./dumpling -u root -h 127.0.0.1 -P 3306 -B mydb -F 256MiB \
-        -o 's3://my-bucket/sql-backup?region=us-west-2'
+        -o 's3://my-bucket/sql-backup'
```
Comment (ran-huang) [inline br/backup-and-restore-storages.md:33]: 如果用户仍然配置了 `region`，dumpling/lightning 的行为是报错还是忽略？

Comment (WangLe1321) [reply br/backup-and-restore-storages.md:33]: 如果 `region` 是对的，则不会报错，如果填错则报错

**Comment by WangLe1321** [comment]
> @WangLe1321 The deleted `region` parameter also exists in other files. Should we delete it from these files as well? image
> 
> image

Yes, I omitted them. I thought that I only needed to modify files in `tidb-lightning` and `br` directories at the beginning. Now, I have removed all the s3 region parameter about br/lightning/dumpling in this project.

File: dm/task-configuration-file-full.md [hunk 2]
```diff
@@ -105,7 +105,7 @@ loaders:                             # load 处理单元的运行配置参数
     pool-size: 16                    # load 处理单元并发执行 dump 处理单元的 SQL 文件的线程数量，默认值为 16，当有多个实例同时向 TiDB 迁移数据时可根据负载情况适当调小该值
 
     # 保存上游全量导出数据的目录。该配置项的默认值为 "./dumped_data"。
-    # 支持配置为本地文件系统路径，也支持配置为 Amazon S3 路径，如: s3://dm_bucket/dumped_data?region=us-west-2&endpoint=s3-website.us-east-2.amazonaws.com&access_key=s3accesskey&secret_access_key=s3secretkey&force_path_style=true
+    # 支持配置为本地文件系统路径，也支持配置为 Amazon S3 路径，如: s3://dm_bucket/dumped_data?endpoint=s3-website.us-east-2.amazonaws.com&access_key=s3accesskey&secret_access_key=s3secretkey&force_path_style=true
```
Comment (ran-huang) [inline dm/task-configuration-file-full.md:108]: @WangLe1321 又改了一个，麻烦看下是否正确

**Review by sunzhaoyang — APPROVED** [review]
LGTM

**Review by 3pointer — APPROVED** [review]
LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**